project_name/urls.py

```diff
@@ -19,8 +19,7 @@
 from django.conf.urls.static import static
 
 urlpatterns = [
-    # Uncomment to add main app urls to root page
-    # path('', include('main.urls')),
+    path('', include('main.urls')),
```

Comment:
If this line is uncommented in more than one branch, sometimes cause conflicts.

Reply:
Then none of the other branches should uncomment it, or only a single one.